src/user/user.controller.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  Body,
+  Controller,
+  Get,
+  HttpException,
+  HttpStatus,
+  Post,
+} from '@nestjs/common';
+import { UserService } from './user.service';
+
+@Controller('user')
+export class UserController {
+  constructor(private userservice: UserService) {}
+
+  @Post('create')
+  createNewUser(
+    @Body() userDto: { id: number; username: string; password: string },
+  ) {
+    return this.userservice.createUser(
```

Comment:
Create user should take only one parameter which is userDto, wenta khod ele enta 3ayzo b2a b3d kda mno